migrations/iql6bkdk.skill-category-is-activated.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE skill ADD COLUMN is_active BOOLEAN NOT NULL DEFAULT true;
+ALTER TABLE skill_category ADD COLUMN is_active BOOLEAN NOT NULL DEFAULT true;
+---
+ALTER TABLE skill DROP COLUMN is_active RESTRICT;
```

Comment:
no need for RESTRICT here--there are no dependencies on this column (foreign keys, constraints, etc)